.github/cloud-samples-tools/test/utils.go

```diff
@@ -0,0 +1,45 @@
+package test
+
+import (
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+// assert fails the test if the condition is false. (https://github.com/benbjohnson/testing)
+func assert(tb testing.TB, condition bool, msg string, v ...interface{}) {
```

Comment:
Ah, yes. The lack of `assert()` (and related functions) in Go's standard libraries. I feel this pain.

We shouldn't have to recreate the wheel. Although we should usually be wary of adding third-party dependencies, here's a tried-and-true library used a lot in industry: https://github.com/stretchr/testify . I recommend using this library instead of creating our own.

Reply:
Is there a way to install this only when testing? Like dev-dependencies in Node? I would like to avoid as much overhead when installing the package during PR checks to reduce latency on tests.

Reply:
Ended up removing the helper functions in favor of manual checks to avoid third party dependencies.